packages/fxa-content-server/app/tests/spec/views/mixins/third-party-auth-mixin.js

```diff
@@ -121,6 +161,46 @@ describe('views/mixins/third-party-auth-mixin', function () {
     assert.isTrue(mockForm.submit.calledOnce);
   });
 
+  it('appleSignIn', async () => {
```

Comment:
Another tiny nit, not sure this needs to be async.

This probably also doesn't matter but on L167, `?orignal_oauth_params` has a typo for `original`